src/aggregate/aggregate_request.c

```diff
@@ -985,10 +985,17 @@ int AREQ_ApplyContext(AREQ *req, RedisSearchCtx *sctx, QueryError *status) {
   RSSearchOptions *opts = &req->searchopts;
   req->sctx = sctx;
 
+  if (isSpecJson(index) && (req->reqflags & QEXEC_F_SEND_HIGHLIGHT)) {
+    QueryError_SetError(
+        status, QUERY_EINVAL,
+        "HIGHLIGHT/SUMMARIZE is not supported on JSON indexes");
```

Comment:
Rephrase nitpick - WDYT?
```suggestion
        "HIGHLIGHT/SUMMARIZE is not supported with JSON indexes");
```

Reply:
OK Change done.